### PR TITLE
feat: remove the covid banner on our main page

### PR DESCRIPTION
**Pull request by paulstey**

### CODE REVIEW

File: .github/workflows/preview_on_pr.yml [hunk 1]
```diff
@@ -59,7 +59,8 @@ jobs:
       with:
         schema: ${{ matrix.values.schema }}
         target: ${{ matrix.values.target }}
-        strict: '1'
+          #Uncomment to disable strict checks 
+          #no-strict: true
```
Comment (mirestrepo) [inline .github/workflows/preview_on_pr.yml:63]: 😂- I'm don't think we should disable the strict checking. People won't know when the yamls are not up to the spec

**Review by mirestrepo — CHANGES_REQUESTED** [review]

File: .github/workflows/preview_on_pr.yml (hunk 1 above)
Comment (paulstey) [reply .github/workflows/preview_on_pr.yml:63]: The API of the workflow changed in v4. The _default_ is now `strict`. If we want non-strict, we need to uncomment the section that I commented out.

Comment (mirestrepo) [reply .github/workflows/preview_on_pr.yml:63]: ah okay! Just remove the commented out lines then!

Comment (paulstey) [reply .github/workflows/preview_on_pr.yml]: Done!

**Comment by paulstey** [comment]
The generation of static content (i.e., the `dist` directory) is failing with the following message:

```Error: error:0308010C:digital envelope routines::unsupported```

[Per this StackOverflow question](https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported), this seems to be a consequence of some security changes related to SSL that happened in Node 17.

**Review by mirestrepo — APPROVED** [review]
@paulstey Let's merge this! Can you push some tiny change to confirm that the build and preview are working as expected since the link expired and we had so much back and forth on this. Thanks!

**Pull request merged**